### ISSUE TITLE
Tutorial lists wrong angular.module name

### ISSUE THREAD

**Issue by herereadthis** [yeoman/yeoman#1349]
On the tutorial at http://yeoman.io/codelab/write-app.html, there's this step:

![screen shot 2014-06-11 at 12 32 45](https://cloud.githubusercontent.com/assets/638189/3247617/1d91bade-f186-11e3-95e1-7db5b9614092.png)

But the steps on the previous pages of the tutorial will set up this in scripts/controllers/main.js:

![screen shot 2014-06-11 at 12 34 22](https://cloud.githubusercontent.com/assets/638189/3247630/43346e26-f186-11e3-826a-5b1fad57e7dd.png)

Which would make sense because index.html shows this:

![screen shot 2014-06-11 at 12 34 43](https://cloud.githubusercontent.com/assets/638189/3247637/5df57048-f186-11e3-8ae0-408a61f636c9.png)

Basically, the tutorial is saying that the name should be **mytodoApp** but it really should be **yeomanTutorialApp** If you run through the tutorial, and keep the name **yeomanTutorialApp**, the todo list will continue to work.
